app/src/main/java/org/mifos/selfserviceapp/presenters/LoginPresenter.java

```diff
@@ -77,6 +77,7 @@ public void login(final String username, final String password) {
         checkViewAttached();
         getMvpView().showProgress();
```

Comment:
Remove this line, Do not need to show Progressbar before validating the username or password.